docs/source/glossary.rst

```diff
@@ -105,6 +105,9 @@ General Terms
     Manta
         Manta is an SV caller. Source for Manta can be found `here <https://github.com/Illumina/manta>`__ [Chen-2016]_
 
+    Strelka
+        Strelka is an SV caller. Source for Strelka can be found `here <https://github.com/Illumina/strelka>`__ [Saunders-2012]_
```

Comment:
I wouldn't say its an SV caller, perhaps saying it is a "SNV and small indel caller" would be more appropriate. Also add a note here that we do not process the SNVs

Reply:
ok I'll change it